src/Serilog.Sinks.Exceptionless/Sinks/Exceptionless/ExceptionlessSink.cs

```diff
@@ -45,6 +45,7 @@ public class ExceptionlessSink : ILogEventSink {
                     config.ServerUrl = serverUrl;
 
                 config.UseInMemoryStorage();
+                config.UseLogger(new SelfLogLogger());
```

Comment:
I don't think this should be included by default. We should add this to the docs, diagnostic messages shouldn't be included by default.